### PR TITLE
footer-a11y-issue

### PR DESCRIPTION
**Pull request by calvin-fung-zopa**

### CODE REVIEW

File: src/components/molecules/ZopaFooter/styles/Heading.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components';
 import ZopaHeading from '../../../atoms/Heading/Heading';
 
-export const Heading = styled(ZopaHeading).attrs({ as: 'h3', size: 'h6' })`
+export const Heading = styled(ZopaHeading).attrs({ as: 'p', size: 'h6' })`
```
Comment (emilfjellstrom) [inline src/components/molecules/ZopaFooter/styles/Heading.tsx:4]: I think this styled component should wrap the `Text` component as the `Heading` should not allow the attribute `as` as `p`

**Review by emilfjellstrom — CHANGES_REQUESTED** [review]

File: src/components/molecules/ZopaFooter/styles/Heading.tsx [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
-import ZopaHeading from '../../../atoms/Heading/Heading';
+import Text from '../../../atoms/Text/Text';
+import { colors } from '../../../../constants';
 
-export const Heading = styled(ZopaHeading).attrs({ as: 'h3', size: 'h6' })`
+export const Heading = styled(Text).attrs({
+  weight: 'bold',
+  color: colors.greyDarkest,
+})`
   text-transform: uppercase;
```
Comment (emilfjellstrom) [inline src/components/molecules/ZopaFooter/styles/Heading.tsx:9]: There's a `capitalize` prop for the `Text` that we can use here instead of the css

**Review by emilfjellstrom — DISMISSED** [review]
LGTM

File: src/components/molecules/ZopaFooter/styles/Heading.tsx (hunk 2 above)
Comment (calvin-fung-zopa) [reply src/components/molecules/ZopaFooter/styles/Heading.tsx]: The capitalised prop has a relationship with the size of the text, hence using that prop would mean that the text can only be small.

**Review by poteirard — APPROVED** [review]

**Pull request merged**